docs/09-security/contingency-plan.md

```diff
@@ -123,35 +123,33 @@ If there is a disruption in the OpsGenie service, all alerts automatically get d
 
 #### JIRA
 
-- **Service:** <https://globalnet.atlassian.net/>
+- **Service:** <https://PROJECT.atlassian.net/>
 - **Status:** <https://twitter.com/JIRA>
 
-There is no direct impact to the platform if a disruption occurs. Primary incident communications will move to the [`#globalnet`](https://civicactions.slack.com/messages/globalnet/) Slack channel.
+There is no direct impact to the platform if a disruption occurs. Primary incident communications will move to the [CivicActions `#general`](https://civicactions.slack.com/) Slack channel.
 
 #### Slack
 
-- **Service:** <https://civicactions.slack.com/messages/globalnet/>
+- **Service:** <https://civicactions.slack.com/>
 - **Status:** <https://status.slack.com/>
 - **Status:** <https://twitter.com/SlackStatus>
-- **Backup:** <https://chat.google.com/> - useful for realtime comms if Slack is down
 
 There is no direct impact to the platform if a disruption occurs.
-Primary incident communications will move to one of (try in order):
+Primary incident communications will move to one of:
 
-- GlobalNET Maintenance Scrum Hangout <https://hangouts.google.com/hangouts/_/civicactions.net/maintenance>
+- IT Zoom: <https://zoom.us/>
+- Google Meet: <https://meet.google.com/>
 - Google Chat: <https://chat.google.com/>
-- IT Zoom: <https://zoom.us/j/865708118> Meeting ID: 865 708 118
-- IT Google Hangout: <https://hangouts.google.com/hangouts/_/civicactions.net/it_is>
 
 #### CPM
 
-- **Service:** <https://gnet-cpm.civicactions.net/signin/>
+- **Service:** <https://cpm.civicactions.net/>
```

Comment:
This URL doesn't exist and we don't use CPM internally or on most projects - we might just remove this section?

Reply:
removed CPM